### ISSUE TITLE
unhashing: sign of a feature can be confusing in case of collisions

### ISSUE THREAD

**Issue by kmike** [TeamHG-Memex/eli5#19]
A follow-up to https://github.com/TeamHG-Memex/eli5/issues/10 and https://github.com/TeamHG-Memex/eli5/issues/18: when deciding if a feature should be in top positive or in top negative features we should take in account sign of the most popular term, e.g. instead of 

`(-)people | considered | approximately   +1.739` (as it is now)

it should be better to show

`people | (-)considered | (-)approximately   -1.739`

**Comment by lopuhin** [comment]
Yeah, that's a great idea! Would make feature report much easier to read.

**Comment by lopuhin** [comment]
See #20 , but now I'm not sure it's sufficient

**Comment by lopuhin** [comment]
Ah, no, it's enough - I was confused because I was getting feature_names myself and forgot about always_signed.

So should be fixed in #20.

**Comment by kmike** [comment]
#20 works great, thanks!
